cmd/lakefs/cmd/root.go

```diff
@@ -49,8 +49,8 @@ func init() {
 }
 
 func validateQuickstartEnv(cfg *config.Config) {
-	if ((cfg.Database.Type != local.DriverName && cfg.Database.Type != mem.DriverName) || cfg.Blockstore.Type != block.BlockstoreTypeLocal) {
-		fmt.Println("quickstart mode can only run with local settings")
+	if (cfg.Database.Type != local.DriverName && cfg.Database.Type != mem.DriverName) || cfg.Blockstore.Type != block.BlockstoreTypeLocal {
+		_, _ = fmt.Fprint(os.Stderr, "\nERROR: quickstart mode can only run with local settings\n")
```

Comment:
Would `FATAL` be more accurate than `ERROR`?

Reply:
Sure

Reply:
@rmoff  Is this what you meant in #6401? Can we close it?

Reply:
![thumbs-up-double-thumbs-up-gif](https://github.com/treeverse/lakeFS/assets/3671582/41e51365-9de7-457b-b257-a601a410c2a5)